.github/workflows/check_wheel_availability.yaml

```
@@ -19,20 +19,32 @@ jobs:
   check_dependency_artifacts:
     name: Check Dependency Artifacts
     runs-on: ${{ matrix.os }}
+    container: ${{ matrix.os == 'arm64' && format('docker://python:{0}', matrix.python-version) || '' }}
     strategy:
       fail-fast: false
       matrix:
         python-version: ['3.7', '3.8', '3.9']
-        os: [macOS-latest, ubuntu-latest]
+        os: [macos-latest, m1, ubuntu-latest, arm64]
```

```suggestion
        os: [macos-latest, m1, ubuntu-latest, [Linux, ARM64]]
```
So that this doesn't end up landing on the native m1 runners